src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -18,10 +19,12 @@
     public static final String DEFAULT_NAME = "Alice Pauline";
     public static final String DEFAULT_PHONE = "85355255";
     public static final String DEFAULT_ADDRESS = "123, Jurong West Ave 6, #08-111";
+    public static final boolean DEFAULT_ARCHIVESTATUS = false;
 
     private Name name;
     private Phone phone;
     private Address address;
+    private ArchiveStatus archiveStatus;
```

Comment:
Perhaps the naming of this field could be changed to isArchived to show that ArchiveStatus can only be true or false ?

Reply:
But I think normally for variable names, we should use a noun. And only for method, we use something starting with a verb, like isArchived. So I think here, archiveStatus should be used (: Thanks!